### PR TITLE
Add button class to button output for styling

### PR DESCRIPTION
**Pull request by jdevalk**
Without a `button` class, styling this element on the frontend is hard :)

### CODE REVIEW

File: blocks/library/button/index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ registerBlockType( 'core/button', {
 		const { url, text, title, align = 'none' } = attributes;
 
 		return (
-			<div className={ `align${ align }` }>
```
Comment (youknowriad) [inline blocks/library/button/index.js]: I would suggest `blocks-button` instead to avoid ambiguity with already defined `button` classes.

Comment (mtias) [inline blocks/library/button/index.js]: We should take a step back and figure out a consistent way for naming these classes across all blocks. Not sure `blocks-{name}` will be enough. Maybe we should use the namespace of the block `wp-core-button`? This is something we want to get right and not change.

Comment (iseulde) [inline blocks/library/button/index.js]: All blocks have a unique attribute like: `div[data-type="core/preformatted"]`.

**Comment by mtias** [comment]
This is superceded by #1381 which adds auto-generated wrapper classes to most blocks.

**Comment by gziolo** [comment]
It's fixed now with #1381. All block have `wp-block-button' as of this change. Closing this one.

**Pull request closed without merging**